File: FrontEnd(Angular 9)/src/app/_shared/model/historique.ts
export class historique {

    id:number;
    soldeIntial:number;
    decouvertIntial:number;
    soldeFinal:number;
    decouvertFinal:number;
    date:string;
    compte:number;


    getId():number{
        return this.id;
    }
    getSoldeIntial():number{
        return this.soldeIntial;
	}
	getDecouvertIntial():number{
        return this.decouvertIntial;
    }
    getSoldeFinal():number{
        return this.soldeFinal;
	}
	getDecouvertFinale():number{
        return this.decouvertFinal;
    }
    getDate():string{
        return this.date;
    }
    getCompte():number{
        return this.compte;
    }
    setId(id:number){
        this.id=id;
	}
	setSoldeIntial(soldeIntial:number){
        this.soldeIntial=soldeIntial;
	}
	setDecouvertIntial(decouvertIntial:number){
        this.decouvertIntial=decouvertIntial;
    }
    setSoldeFinal(soldeFinal:number){
        this.soldeFinal=soldeFinal;
	}
	setDecouvertFinal(decouvertFinal:number){
        this.decouvertFinal=decouvertFinal;
    }
    setDate(date:string){
        this.date=date;
    }
    setCompte(compte:number){
        this.compte=compte;
    }
}

File: BackEnd(Java -jersy)/CompteBancaireWEBAPI/src/model/historique.java
package model;

import java.sql.Date;

public class historique {

 private int id;
 private double soldeIntial;
 private double decouvertIntial;
 private double soldeFinal;
 private double decouvertFinale;
 private Date date;
 private int compte;

 
public historique() {
	
}



public int getId() {
	return id;
}


public void setId(int id) {
	this.id = id;
}


public double getSoldeIntial() {
	return soldeIntial;
}


public void setSoldeIntial(double soldeIntial) {
	this.soldeIntial = soldeIntial;
}


public double getSoldeFinal() {
	return soldeFinal;
}


public void setSoldeFinal(double soldeFinal) {
	this.soldeFinal = soldeFinal;
}


public Date getDate() {
	return date;
}


public void setDate(Date date) {
	this.date = date;
}


public int getCompte() {
	return compte;
}


public void setCompte(int compte) {
	this.compte = compte;
}



public double getDecouvertIntial() {
	return decouvertIntial;
}



public void setDecouvertIntial(double decouvertIntial) {
	this.decouvertIntial = decouvertIntial;
}



public double getDecouvertFinale() {
	return decouvertFinale;
}



public void setDecouvertFinale(double decouvertFinale) {
	this.decouvertFinale = decouvertFinale;
}


 
	
}
